### PR TITLE
Feature/add leds switches

### PR DESCRIPTION
**Pull request by Warukira**
## Description

Add status LEDs and switches functionality.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Screenshots
N/A

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by chegejames — CHANGES_REQUESTED** [review]
@Warukira following our call this morning, please let me know when changes are ready for review.

**Comment by Warukira** [comment]
@chegejames , the requested changes have been effected. Kindly review them.

File: airrohr-firmware/airrohr-firmware.ino [hunk 1]
```diff
@@ -3691,10 +3704,36 @@ String fetchSensorGPSFromAtmega(){
 		}
 	}
 
+	// Obtain GPS send_time from RTC
+	char buf3[40];
+	DateTime now = rtc.now();
+	sprintf(buf3, "%04d-%02d-%02dT%02d:%02d:%02dZ", now.year(), now.month(), now.day(), now.hour(), now.minute(), now.second());
+	timestamp = buf3;
+
+	pcf8575.digitalWrite(P0, HIGH); // turn GPS status led on for 3 seconds
```
Comment (chegejames) [inline airrohr-firmware/airrohr-firmware.ino]: @Warukira I believe we talked about adding this to a function

File: airrohr-firmware/airrohr-firmware.ino [hunk 2]
```diff
@@ -4427,12 +4479,18 @@ static void powerOnTestSensors() {
...
+		pcf8575.digitalWrite(P5, HIGH);
+		delay(2000);
+		pcf8575.digitalWrite(P5, LOW);
 	}
 
 	if (cfg::rtc_read) {
 		rtc.begin();
 		debug_outln_info(F("Read Time from RTC..."));
 		init_RTC();
+		pcf8575.digitalWrite(P2, HIGH);
```
Comment (chegejames) [inline airrohr-firmware/airrohr-firmware.ino]: @Warukira can we also have this during power on test?

**Review by chegejames — CHANGES_REQUESTED** [review]

File: airrohr-firmware/airrohr-firmware.ino (hunk 2 above)
Comment (Warukira) [reply airrohr-firmware/airrohr-firmware.ino]: @chegejames , excuse me if I'm getting this wrong, but it currently is in the power on test.

File: airrohr-firmware/airrohr-firmware.ino (hunk 1 above)
Comment (Warukira) [reply airrohr-firmware/airrohr-firmware.ino]: @chegejames , this function has since been added.

**Review by chegejames — APPROVED** [review]

**Pull request merged**